usb/backend/openusb.py

```diff
@@ -491,7 +491,12 @@ def _check(ret):
         ret = ret.value
 
     if ret != 0:
-        raise USBError(_lib.openusb_strerror(ret), ret, _openusb_errno[ret])
+        if ret == OPENUSB_NOT_SUPPORTED:
+            raise NotImplementedError(_lib.openusb_strerror(ret))
```

Comment:
`NotImplementedError`
-> _breaking_ change? If users caught it previously as USBError it will now raise an exception.